### PR TITLE
New version section in .plug file.

### PR DESCRIPTION
**Pull request by gbin**
This deprecated but still support the old vay.

The new section has the format:
[Errbot]
Min=x.y.z-blah
Max=X.Y.Z-blah

If no section exists, assume that any version of errbot is compatible.
Min or Max can also be individually omited which simply relax the
constrain.

### CODE REVIEW

**Comment by gbin** [comment]
@ifigotin available for a review on this one ?

File: docs/user_guide/plugin_development/plugin_compatibility_settings.rst [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+.. code-block:: ini
+
+    [Core]
+    Name = MyPlugin
+    Module = myplugin
+
+    [Documentation]
+    Description = my plugin
+
+    [Errbot]
```
Comment (ifigotin) [inline docs/user_guide/plugin_development/plugin_compatibility_settings.rst]: Is this a good section name ('Errbot') for describing plugin version? Why not just [Plugin-Version] or something like that.

Comment (gbin) [inline docs/user_guide/plugin_development/plugin_compatibility_settings.rst]: It is errbot version :)

**Comment by ifigotin** [comment]
Looks fine to me.

File: errbot/plugin_manager.py [hunk 2]
```diff
@@ -86,6 +89,95 @@ def check_dependencies(path):
...
+        except NoOptionError:
+            log.debug('Plugin %s has no Min Option in [Errbot] section. '
+                      'Assuming this plugin is running on this Errbot'
+                      'version as min version.', name)
+            min_version = VERSION
+
+        try:
+            max_version = config.get("Errbot", "Max")
+        except NoOptionError:
+            log.debug('Plugin %s has no Min Option in [Errbot] section. '
```
Comment (zoni) [inline errbot/plugin_manager.py]: Typo slipped through. :) This should mention 'Max', not 'Min'.

Comment (gbin) [inline errbot/plugin_manager.py]: Thx @zoni ! fixed in 5d2c8527bc3ea6ac5b213223c7bf60caf149c571 directly on master

**Pull request merged**